### PR TITLE
Makes spray cans recyclable again

### PR DESCRIPTION
**Pull request by eLeCtrOssSnake**
## About The Pull Request

Makes spray cans able to be recycled again, after some PR had made them also paint storages, which made them ignore autolathe recycling.
You can now recycle a spray can when the cap is off, and paint autolathe when the cap is off.

## Why It's Good For The Game

Speaks for itself.

## Changelog
:cl:
tweak: make spray cans recyclable again, while preserving the ability to paint autolathe
/:cl:

### CODE REVIEW

**Comment by PowerfulBacon** [comment]
Is capped should really just call ..() anyway, since you can't put it in backpacks or anything like that

**Comment by eLeCtrOssSnake** [comment]
It will supress “Take the cap off first” message then.

**Comment by eLeCtrOssSnake** [comment]
You can put it in a backpack when you’re opening the storage itself, not clicking on the backpack. I think that’s the intended behaviour.

**Comment by PowerfulBacon** [comment]
Yea but if you fill up 1 row of a backpack, it is impossible to put a spray can in it without pressing e

**Comment by eLeCtrOssSnake** [comment]
Is this some kind of an MRP struggle/problem that I am too good to understand? Like yea, that is how it works.
Though if you want me to change that just say it straight up, because I don’t really know do you want me to change it or not.

File: code/game/objects/items/crayons.dm [hunk 1]
```diff
@@ -627,6 +627,9 @@
 	if(!proximity)
 		return
 
+	if(is_capped && istype(target, /obj/machinery/modular_fabricator/autolathe))
```
Comment (Archanial) [inline code/game/objects/items/crayons.dm]: you check capped two lines bellow

Comment (eLeCtrOssSnake) [reply code/game/objects/items/crayons.dm]: Yes, that is intended

Comment (eLeCtrOssSnake) [reply code/game/objects/items/crayons.dm]: I can't merge these two checks.

Comment (eLeCtrOssSnake) [reply code/game/objects/items/crayons.dm]: Wait I actually can.

File: code/game/objects/items/crayons.dm [hunk 2]
```diff
@@ -628,7 +628,10 @@
 		return
 
 	if(is_capped)
-		to_chat(user, "<span class='warning'>Take the cap off first!</span>")
+		if(istype(target, /obj/machinery/modular_fabricator/autolathe))
+			. = ..()
+		else
+			to_chat(user, "<span class='warning'>Take the cap off first!</span>")
 		return
```
Comment (utilbird) [inline code/game/objects/items/crayons.dm]: More readable, marginally faster
```suggestion
		if(istype(target, /obj/machinery/modular_fabricator/autolathe))
			return ..()
		else
			to_chat(user, "<span class='warning'>Take the cap off first!</span>")
			return
```

**Comment by eLeCtrOssSnake** [comment]
Doesn’t seem more readable to me, but hey, you’re a maintainer, I listen.

**Review by utilbird — APPROVED** [review]

**Pull request merged**